build/rollup-config.js

```diff
@@ -24,6 +24,27 @@ const config = {
 	],
 	plugins: [
 		release ? json() : rollupGitVersion(),
+		{
+			name: 'copy-leaflet-assets',
+			generateBundle() {
+				const fileNames = [
+					'leaflet.css',
+					'images/logo.svg',
+					'images/layers.png',
+					'images/layers.svg',
+					'images/layers-2x.png',
+					'images/marker-icon.png',
+					'images/marker-icon.svg',
+					'images/marker-icon-2x.png',
+					'images/marker-shadow.png',
+					'images/marker-shadow.svg',
```

Comment:
If we switch to SVG, I think we should not provide `.png` images anymore. The path for the icons like `layers.png` needs to be changed in the code too.

```suggestion
					'images/layers.svg',
					'images/marker-icon.svg',
					'images/marker-shadow.svg',
```